src/hcf-release/jobs/configure-scf/spec

```diff
@@ -48,3 +48,10 @@ properties:
   cf_mysql.mysql.port:
     description: 'Port used by the service broker and bound applications to reach the service'
     default: 3306
+  nfsbroker.url:
+    description: 'Internal URL of the Persi NFS broker'
```

Comment:
FWIW, you don't actually need to quote values when they're unambiguously strings.